project/Dependencies.scala

```diff
@@ -103,6 +103,9 @@ object Dependencies {
 
     val alpakkaKafkaTestkit =
       "com.typesafe.akka" %% "akka-stream-kafka-testkit" % Versions.alpakkaKafka % allTestConfig
+
+    // FIXME remove when updating to AKka HTTP version without dependency to ssl-config-core
+    val sslConfig = "com.typesafe" %% "ssl-config-core" % "0.6.1" % allTestConfig
```

Comment:
hm, I think this works for the grpc-tests, but we will have to add it to samples until we a compatible release of Akka HTTP/gRPC. Might be better that we add it as a real dependency in akka-projection-grpc until we have that in place?

Reply:
Let's add as deo to akka-projection-grpc for now 👍

Reply:
I'll do the Akka update in #991 first, and then keep this PR focused on SerializedEvent